modules/s3/main.tf

```diff
@@ -0,0 +1,40 @@
+## Local Variables
+locals {
+  orca_bucket_names  = [for k, v in var.buckets : v.name if v.type == "orca"]
+}
+
+resource "aws_s3_bucket" "reports_bucket" {
```

Comment:
I think we can also add an `aws_s3_bucket_policy` resource that will be attached to this bucket if we want to make it more restricted

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_policy

Reply:
So you want something where even requests within the same account are denied unless they are on a whitelist? Feels like a bit of overkill for a non-public-facing resource.

Reply:
looking at our other non-public-facing buckets in sandbox, looks like they don't have a bucket policy in them either. So I guess we don't need a policy right now.